src/components/AddKpiEra/AddKpiEra.vue

```diff
@@ -231,25 +249,36 @@ export default {
     kpiDescription: sync("adminKPI/kpiDescription"), //v-model
     eraHeading: sync("adminKPI/eraHeading"), //v-model
     eraDescription: sync("adminKPI/eraDescription"), //v-model
-    searchField: sync("adminKPI/searchField") //v-model
+    searchField: sync("adminKPI/searchField"), //v-model
+    getAllMember: sync("allMember/allMember")
   },
   mounted() {},
   data() {
     return {
       showKpiform: -1,
-      showEraform: -1
+      showEraform: -1,
+      alertIndex: -1,
+      active: false
     };
   },
   methods: {
+    getAllMembers() {
+      this.getAllMembers_();
+    },
+    getAllMembers_: call("allMember/getAllMember"),
     getProfile: call("profile/getProfile"),
     api_AddKpi: call("adminKPI/addKpi"),
     api_delKpi: call("adminKPI/delKpi"),
     api_updateKpi: call("adminKPI/updateKpi"),
+    api_deleteOnce: call("adminKPI/deleteOnce"),
     inputHandler(e) {
       if (e.keyCode === 13 && !e.shiftKey) {
         e.preventDefault();
       }
     },
+    closeAlert() {
+      this.alertIndex = -1;
+    },
     get_profile: function() {
       this.getProfile({
         Authorization: localStorage.getItem("authenticated")
```

Comment:
i have mentioned this 100 times. i don't want to see this anywhere except axios global